doc/whats_new/v0.20.rst

```diff
@@ -12,6 +12,18 @@ Version 0.20.2
 This is a bug-fix release with some minor documentation improvements and
 enhancements to features released in 0.20.0.
 
+Changed models
+--------------
+
+The following estimators and functions, when fit with the same data and
+parameters, may produce different models from the previous version. This often
+occurs due to changes in the modelling logic (bug fixes or enhancements), or in
+random sampling procedures.
+
+- :mod:`sklearn.neighbors` when ``metric=='jaccard'`` (bug fix)
+- :mod:`sklearn.metrics.pairwise` when ``metric=='jaccard'` and Scipy >= 1.2.0
```

Comment:
this is not related to changes in scikit-learn so remove?

Reply:
This PR was updated to remove the second item regarding `sklearn.metrics.pairwise`.